bdio-tinkerpop/src/main/java/com/blackducksoftware/bdio2/tinkerpop/sqlg/SqlgBlackDuckIoReader.java

```diff
@@ -187,7 +187,10 @@ public SqlgNodeAccumulator addNode(Map<String, Object> node) {
                 node.keySet().forEach(k -> schema.put(k, null));
                 Object outVertexId = node.get(JsonLdConsts.ID);
                 identifierTypes.computeIfAbsent(outVertexLabel, SqlgBlackDuckIoReader::newIdentifierSet).put(outVertexId);
-                getNodeEdges(node, (edgeLabel, inVertexId) -> edges.put(EdgeKey.of(edgeLabel, outVertexLabel), Pair.of(outVertexId, inVertexId)));
+                getNodeEdges(node, (edgeLabel, inVertexId) -> {
```

Comment:
Looks like this change does nothing, except reformatting the line.

Reply:
Yes, just reformatted to add any log if required   for debugging purpose